### PR TITLE
Update UUFO3P

### PR DESCRIPTION
**Pull request by Superfly-Johnson**
Really small change, added the crc for 3.3.6 and added NVAC as a recommended addition.

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -664,6 +667,10 @@ plugins:
       - Stats
       - Text
     clean:
+    # 3.3.6
+      - crc: 0x441BC38C
+        util: 'FO3Edit v4.0.4c'
+    # 3.3.3
       - crc: 0x514A0671
         util: 'FO3Edit v4.0.4c'
```
Comment (MacSplody) [inline masterlist.yaml]: Just remove the old cleaning data, we only keep the latest version.
```suggestion
      - crc: 0x441BC38C
        util: 'FO3Edit v4.0.4c'
```

File: masterlist.yaml [hunk 2]
```diff
@@ -601,9 +601,12 @@ plugins:
       - <<: *alsoUseX
         subs: [ '[lStewieAl''s Tweaks](https://www.nexusmods.com/fallout3/mods/23561/)' ]
         condition: 'not file("FOSE/Plugins/fose_stewie_tweaks.dll") and (version("../fose_1_7.dll", "0.1.3.2", >=) or version("../fose_1_7ng.dll", "0.1.3.2", >=))'
+      - <<: *alsoUseX
+        subs: [ '[NVAC - New Vegas Anti Crash](https://www.nexusmods.com/newvegas/mods/53635/)' ]
+        condition: 'not file("FOSE/Plugins/nvac.dll") and (version("../fose_1_7.dll", "0.1.3.2", >=) or version("../fose_1_7ng.dll", "0.1.3.2", >=))'
```
Comment (MacSplody) [inline masterlist.yaml:606]: I'm not sure about this one tbh.
I know they recommend it, but there are still users reporting crashes when using it.

```yaml
      - <<: *alsoUseX
        subs: [ '[NVAC - New Vegas Anti Crash](https://www.nexusmods.com/newvegas/mods/53635/)' ]
        condition: 'not file("FOSE/Plugins/nvac.dll") and (version("../fose_1_7.dll", "0.1.3.2", >=) or version("../fose_1_7ng.dll", "0.1.3.2", >=))'
```

Comment (Superfly-Johnson) [reply masterlist.yaml:606]: I personally use NVAC in fallout 3 and find it helping, personally I'd go further and add two other FOSE plugins but they aren't directly recommended by the UUFO3P project

Comment (MacSplody) [reply masterlist.yaml:606]: Okay i'll allow it, but if enough people complain about it, we might have to remove it.

**Review by MacSplody — APPROVED** [review]

**Pull request merged**